.travis.yml

```diff
@@ -1,7 +1,10 @@
 sudo: false
 language: node_js
-script: make deps && make dist
+script:
+  - make deps && make dist
+  - if [[ $(node --version) = v6.* ]]; then make -C fluent-syntax test-without-babel-polyfill; fi
```

Comment:
Any reason to only test this for v6 and not for v8 as well?

Reply:
No reason, except that I fixated on Node 6 :)